MobileCenterUpdates/MobileCenterUpdates/MSUpdates.h

```diff
@@ -3,7 +3,7 @@
 @interface MSUpdates : MSServiceAbstract
```

Comment:
We are going to use pre-define format so it is not necessary. I'm make a relevant change for this one.